analyses/molecular-subtyping-HGG/run-molecular-subtyping-HGG_003_008.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
My guess is that this is something you were working with locally having to do with PNOC 003 and 008 and that it should not be added here?

Reply:
yeahh I committed the wrong files here by mistake.. such a pain 😅

Reply:
:P